17457 ensure fgdb libraries are in rpm and update mapserver version

robertvilla opened the pull request:
Updated mapserver version and put FGDB libraries in the correct place for inclusion in the RPM (option "3" from the issue comment #7)

Redmine issue: https://redmine.edina.ac.uk/issues/17457

agxs commented on gdal-el7/Dockerfile:
```diff
@@ -53,7 +53,8 @@ WORKDIR "/root/gdal-${version}"
 RUN make && make install DESTDIR=/tmp/gdal && make clean && ldconfig
 
-RUN cp /usr/local/FileGDB_API-64/lib/libFileGDBAPI.so /usr/lib && cp /usr/local/FileGDB_API-64/lib/libfgdbunixrtl.so /usr/lib && ldconfig
+RUN cp /usr/local/FileGDB_API-64/lib/libFileGDBAPI.so /tmp/gdal/usr/lib
```
Having these on 1 run is good. Use \ to break them up if you need to make the dockerfile read better.
Each run creates a new filesystem so its good to minimise that where possible.

robertvilla commented:
I've changed the way the fildGDB libraries are installed (slightly neater than copying directly to /usr/lib). Also changed mapserver a little, I had previously listed -devel dependencies which are not required.

agxs commented:
Ah yes I didn't notice the -devel deps.
